### ISSUE TITLE
[Feature Request] Option to disable "Edit mode by double click on Html preview"

### ISSUE THREAD

**Issue by sakkamade** [dvorka/mindforger#1310]
Hello.

My suggestion is to implement an option which would disable the "Double click on preview to transition in edit mode" function, because it can be truly annoying at times.
 
For example, creating the button
```html
<button type="button" onclick="document.getElementById('id').style.display='block'">show</button>
```
and double clicking it in preview, sends me straight into edit mode.<br /> 

Truly inconvenient.

**Comment by roshiajin-kunai** [comment]
The same happens if after clicking a<br /> `<button type="button" onclick="document.getElementById('8').style.display='none',document.getElementById('9').style.display='block'">hide</button>`  <br />button, and it's place takes another<br /> `<button type="button" onclick="document.getElementById('8').style.display='block',document.getElementById('9').style.display='none'">show</button>`<br />button.<br />If I'd on press them too quickly, I'll turn on the edit mode, again. <br />Yes, maybe my code is bad, but it works, so I'm fine.

By the way. This program is very nice and useful for me. Thank you very much.

**Comment by dvorka** [comment]
Fixed by 3939ae2 - will be released in 1.53.0

![double](https://user-images.githubusercontent.com/2090143/142276586-61384aab-50df-481b-bd89-6a44ccb09e67.png)

**Comment by sakkamade** [comment]
Thank you!

**Comment by sakkamade** [comment]
So, tested the release version 1.53, and found issue: Unchecking "double click view to edit" also disables the "Edit" button under the view, this one
![image](https://user-images.githubusercontent.com/62730422/148149311-b002dee8-6e15-4310-9c1d-d69b2a29b1f4.png)

In other words, nothing happens when I uncheck "double click view to edit" and click that Edit button.

**Comment by dvorka** [comment]
@sakkamade apologies - 🐞 you described is reproducible on b45ebbc1497c0d6a32e31ce227b9aa71762af174 - reopenning.

**Comment by dvorka** [comment]
`Edit` button fixed by cd34edf4c09edc2449f83c98ccb9a22d58abf347 - will be released with 1.54.0

@sakkamade not sure whether it would help, but if it's not complicated, then you can use branch `dev/1.54.0` (and commit ^) to try the fix OR wait for the next release (will be fix release).

**Comment by sakkamade** [comment]
Sorry, not sure what you meant by "(and commit)" (if I could, I would have already).

But I compiled MindForger with this 37ca69877aff5d73fe447bb15f5b340106183f4f commit latest, and it appears to be fixed.
Thank you very much!

Can be closed?
